cgroups/systemd/apply_systemd.go

```diff
@@ -212,6 +213,17 @@ func (m *Manager) Apply(pid int) error {
 
 	m.Paths = paths
 
+	cpuShares, err := ioutil.ReadFile(path.Join(m.Paths["cpu"], "cpu.shares"))
```

Comment:
Probably better to create temp variable for `strings.TrimSpace(string(cpuShares))` and then use it in condition and error